docs/running-on-yarn.md

```diff
@@ -852,8 +852,8 @@ The following extra configuration options are available when the shuffle service
   <td><code>spark.shuffle.service.db.backend</code></td>
   <td>LEVELDB</td>
   <td>
-    To specify the kind of disk-base store used in shuffle service state store, supports `LEVELDB` and `ROCKSDB` now 
-    and `LEVELDB` as default value. 
+    When work-preserving restart is enabled in YARN, this is used to specify the disk-base store used 
+    in shuffle service state store, supports `LEVELDB` and `ROCKSDB` with `LEVELDB` as default value. 
     The original data store in `LevelDB/RocksDB` will not be automatically convert to another kind of storage now.
```

Comment:
`convert` -> `converted` ?

Reply:
+1 for @mridulm comment. And, could you add additional description about what happens at the runtime when the the store types are mismatched. It's deleted and recreated, right?

Reply:
The old one will not be deleted, but the new one will be created. When the store type is switched, the directory name will change, for example, from `registeredExecutors.ldb` to `registeredExecutors.rdb`, `YarnShuffleService` will create `registeredExecutors.rdb` if it not exists, but `YarnShuffleService` did not know that `registeredExecutors.ldb` existed, so it will not be deleted

Reply:
Add `The original data store will be retained and the new type data store will be created when switching storage types.` Is that ok ?

Reply:
@dongjoon-hyun @mridulm Automatic data format conversion may be a useful feature. I think it is more friendly for migrating stock users to use new features. I have filed a Jira SPARK-40464 and will promote its completion if necessary.

Reply:
> @dongjoon-hyun @mridulm Automatic data format conversion may be a useful feature. I think it is more friendly for migrating stock users to use new features. I have filed a Jira [SPARK-40464](https://issues.apache.org/jira/browse/SPARK-40464) and will promote its completion if necessary.

also cc @panbingkun , what I discussed with you offline yesterday